File: src/Management/Models/Course.cs
using System.ComponentModel.DataAnnotations;

namespace Management;

public class Course
{
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        public string Description { get; set; }

        public string Category { get; set; }

        public string Instructor { get; set; }

        public DateTime DateCreated { get; set; }

        public DateTime DateEnded { get; set; }

        public ICollection<StudentCourse> StudentCourses { get; set; }
    }

File: web/src/components/Course.js
import React from 'react';

const Course = (props) => {
    return (
        <tr>
            {/* todo: rm id */}
            {/* <td>{props.id}</td> */}
            <td>{props.name}</td>
            <td>{props.description}</td>
            <td>{props.children}</td>
        </tr>
    );
};

export default Course;
